lib/teleterm/clusters/cluster.go

```diff
@@ -81,14 +83,19 @@ func (c *Cluster) Connected() bool {
 // and enabled enterprise features. This method requires a valid cert.
 func (c *Cluster) GetWithDetails(ctx context.Context) (*ClusterWithDetails, error) {
 	var (
-		pingResponse  proto.PingResponse
-		caps          *types.AccessCapabilities
-		authClusterID string
-		acl           *api.ACL
-		user          types.User
+		authPingResponse proto.PingResponse
+		caps             *types.AccessCapabilities
+		authClusterID    string
+		acl              *api.ACL
+		user             types.User
 	)
 
-	err := AddMetadataToRetryableError(ctx, func() error {
+	clusterPingResponse, err := c.clusterClient.Ping(ctx)
```

Comment:
If I'm not mistaken, `clusterClient.Ping` returns the proxy server version, while `authClient.Ping` returns the auth server version. 

From reading the [component compatibility article](https://goteleport.com/docs/management/operations/upgrading/#component-compatibility), I understood that the "proxy version" is the version that other resources depend on. 

But perhaps I'm overthinking it and we could just use the auth server version that we have?

Reply:
Ah, I totally forgot that there would be two different versions available. In that case I think you're right, the proxy version is the one we should inspect – both Connect and the agent connect to the auth server through the proxy and not directly to the auth server.

In that case, should we rename the field to be `proxyVersion` instead of `serverVersion`? With `serverVersion`, it's not clear whether that's the version of the proxy or the auth server.

Reply:
Renamed to `proxyVersion`.

Reply:
Fantastic. 😎